### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by Tolexia**

### CODE REVIEW

File: src/Controller/EventController.php [hunk 1]
```diff
@@ -83,8 +83,14 @@ public function invite(Event $event, User $user, EntityManagerInterface $entityM
      */
     public function show(Event $event): Response
     {
+        $participants = $event->getParticipants();
+        $isParticipant = false;
+        if (in_array($this->getUser(), $participants->getValues())) {
+            $isParticipant = true;
+        }
```
Comment (rocambille) [inline src/Controller/EventController.php]: try this:

    $isParticipant = in_array($this->getUser(), $participants->getValues());

**Review by rocambille — CHANGES_REQUESTED** [review]

File: templates/event/show.html.twig [hunk 2]
```diff
@@ -53,5 +53,20 @@
         {% endfor %}
         </ul>
         {% endif %}
-    </div>
+        {% if isParticipant %}
```
Comment (rocambille) [inline templates/event/show.html.twig]: looks like:

    {% if event.hasParticipant(app.user) %}

does'nt it?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Controller/EventController.php (hunk 1 above)
Comment (Tolexia) [reply src/Controller/EventController.php]: Done ! Smooth, should have thought about that :)

File: templates/event/show.html.twig (hunk 2 above)
Comment (Tolexia) [reply templates/event/show.html.twig]: Totally, gonna change this

File: templates/event/show.html.twig [hunk 3]
```diff
@@ -53,5 +53,20 @@
...
+                    <div class="friend-show-picture-background">
+                        <img class="friend-picture" src="{{ asset('/uploads/' ~ app.user.avatar) }}">
+                    </div>
+                    <p class="friend-name">{{app.user.firstname ~ ' ' ~ app.user.lastname}}</p>
+                    <form class="friend-invite" action="{{ path('event_user_toggle', {"event": event.id, "user": app.user.id}) }}" method="post">
+                        <button class="friend-invite-button {{ event.hasParticipant(app.user) ? "remove" : 'add' }}" type="submit">{{ event.hasParticipant(app.user) ? "-" : '+' }}</button>
+                    </form>
+                </div>
+            </li>
+        </div>
```
Comment (rocambille) [inline templates/event/show.html.twig]: which `div` is closed by this tag?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: templates/event/show.html.twig (hunk 3 above)
Comment (Tolexia) [reply templates/event/show.html.twig]: none

**Pull request merged**